### DISCUSSION TITLE
Moving containers inside a file from one location to another?

### DISCUSSION THREAD

**Discussion by jthermiz** [NeurodataWithoutBorders/helpdesk#21] [Q&A]
I wrote a script that reorganizes the acquisition container like this:

**Old file**
acquisition
    Raw
        Eseries1
        Eseries2
        ...

**New file**
acquisition
        Eseries1
        Eseries2
        ...

However, this implementation seems to just link the new electrical series to the old file. So when I move or destroy the old file, I won't be able to access the data in the new file. Is there an easy way to make a hard copy of the Electrical Series (ie without creating a new Electrical Series class from the data primitives?)

**Comment by rly** [comment]
Unfortunately, moving containers inside a file from one location to another is not currently supported. Although PyNWB supports taking datasets read from one file and writing them to a new file, taking entire containers that are read from one file and adding/writing them to a new file is not currently supported. 

I've created an issue ticket on this here: https://github.com/hdmf-dev/hdmf/issues/691

For now, you can work around the issue using the hacky last section of the following code: https://gist.github.com/rly/65f54f0c15a4ce0444b474fd0c1732ed
